src/Workspaces/Core/MSBuild/MSBuild/Build/ProjectBuildManager.cs

```diff
@@ -87,7 +87,9 @@ public ProjectBuildManager(ImmutableDictionary<string, string> additionalGlobalP
 
                 using (var stream = FileUtilities.OpenAsyncRead(path))
                 using (var readStream = await SerializableBytes.CreateReadableStreamAsync(stream, cancellationToken).ConfigureAwait(false))
+#pragma warning disable CA3075 // Insecure DTD processing in XML
```

Comment:
Why is this safe to skip this one? I thought this was a security rule and thus we'd have to follow it.

Reply:
Agreed.

Reply:
In fact, this seems like the only really important one, as the others are in test code.

Reply:
Tagging @dotpaul who has owns this security analyzer and has recently looked at false positives around it - https://github.com/dotnet/roslyn-analyzers/issues/1591#issuecomment-443960542

Reply:
From the thread on that issue, seems like default setting for `XmlReader.Create` is secure and this is false positive from the analyzer. I'd still let @dotpaul confirm.

Reply:
The fix is in v2.6.3.

Reply:
Guess I can't comment on lines not in the diff, but &lt;RoslynDiagnosticsNugetPackageVersion&gt; in Versions.prop is 2.6.2-beta2.

Reply:
This is indeed a false positive, not necessarily because of the defaults for `XmlReader.Create` but because `s_xmlReaderSettings` is defined correctly according to the security guidelines:
https://github.com/dotnet/roslyn/blob/bae1454a81c62db5820ced24e7885bd07c3a471b/src/Workspaces/Core/MSBuild/MSBuild/Build/ProjectBuildManager.cs#L19-L23

I was planning on verifying if a new version of the analyzer had fixed this today so thank you for doing that for me while I was asleep :)

Reply:
Oops, sorry, I was thinking of a different overload.  I don't think that case has been fixed yet.

Reply:
Can we make sure there is a comment on the suppression that explains why it's safe to suppress?